Lil-Peep/src/styles/modules/site-footer.scss

```diff
@@ -1,226 +1,227 @@
 footer {
-  background-color: #ffffff;
+  background-color: $color-body;
   border-top-color: rgba(28, 26, 26, 0.1);
   border-top-style: solid;
   border-top-width: 1px;
   bottom: 0;
   display: flex;
   flex-direction: column;
+  margin: auto;
   margin-top: 15px;
-  width: 100%;
+  max-width: 2250px;
+  width: 80%;
 }
 
 .footer-content {
   margin: 0 40px;
-}
-
-.social-media-container {
-  align-items: center;
-  display: flex;
-  justify-content: center;
-  width: 100%;
 
-  .social-media-box {
+  .social-media-container {
+    align-items: center;
     display: flex;
-    flex: 1;
-    text-align: center;
+    justify-content: center;
+    width: 100%;
 
-    .social-media {
-      color: #72808C;
-      display: block;
-      font-family: 'Lato', sans-serif;
-      font-size: 18px;
-      line-height: 21px;
+    .social-media-box {
+      display: flex;
+      flex: 1;
+      text-align: center;
 
-      p {
-        @media #{$breakpoint-mobile} {
-          font-size: 14px;
-          text-align: center;
-          width: 100%;
+      .social-media {
+        color: $color-theme-lilpeep;
+        display: block;
+        font-family: $lato-font;
+        font-size: 18px;
+        line-height: 21px;
+
+        p {
+          @media #{$breakpoint-mobile} {
+            font-size: 14px;
+            text-align: center;
+            width: 100%;
+          }
         }
       }
-    }
 
-  
+      .sm {
+        align-items: center;
+        border: 2px solid $color-border-box;
+        border-radius: 3px;
+        display: inline-flex;
+        height: 35px;
+        justify-content: center;
+        width: 100px;
 
-    .sm {
-      align-items: center;
-      border: 2px solid #72808c;
-      border-radius: 3px;
-      display: inline-flex;
-      height: 35px;
-      justify-content: center;
-      width: 100px;
+        img {
+          max-height: 20px;
+        }
 
-      img {
-        max-height: 20px;
-      }
+        @media #{$breakpoint-desktop} {
+          border: 2px solid $color-border-box;
+          border-radius: 3px;
+          height: 25px;
+          width: 60px;
+        }
 
-      @media #{$breakpoint-desktop} {
-        border: 2px solid #72808c;
-        border-radius: 3px;
-        height: 25px;
-        width: 60px;
+        @media #{$breakpoint-mobile} {
+          display: inline-flex;
+          height: 35px;
+          width: 30px;
+        }
       }
 
-      @media #{$breakpoint-mobile} {
-        display: inline-flex;
-        height: 35px;
-        width: 85px;
-      }
-    }
+      .twt {
+        margin: 0 7px 0 7px;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0 7px` instead of `0 7px 0 7px`